jubatus/core/storage/labels.cpp

```diff
@@ -0,0 +1,118 @@
+// Jubatus: Online machine learning framework for distributed environment
+// Copyright (C) 2016 Preferred Networks and Nippon Telegraph and Telephone Corporation.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License version 2.1 as published by the Free Software Foundation.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+#include "labels.hpp"
+
+#include <string>
+
+#include "jubatus/util/concurrent/lock.h"
+
+namespace jubatus {
+namespace core {
+namespace storage {
+
+labels::labels() {
+}
+
+labels::~labels() {
+}
+
+bool labels::set(const std::string& label) {
```

Comment:
`set` sounds unnatural, as this data structure is a **counter**.
I think `add` is better.